tsconfig.json

```diff
@@ -1,8 +1,9 @@
 {
     "compilerOptions": {
         "target": "es5",
+        "outDir": "lib/",
         "module": "commonjs",
-        "noEmitHelpers": true,
+        "noEmitHelpers": false,
```

Comment:
I think before this was emitted in the global scope, now that it's an ES6 module it gets it's own scope :)